epoch1/cuda/ee_mumu/src/Makefile

```diff
@@ -22,6 +22,9 @@ ifneq ($(wildcard $(CUDA_HOME)/bin/nvcc),)
   CUINC = -I$(CUDA_HOME)/include/
 endif
 
+# Set the host compiler for nvcc
+override NVCC := $(NVCC) -ccbin $(subst ccache ,,$(CXX))
```

Comment:
And the same here. I don't know which of the two triggered the error, but it should be the same solution.